Makefile

```diff
@@ -96,7 +101,7 @@ _build_containerized:
 	@if [ -z '$(CTR_CMD)' ] ; then echo '!! ERROR: containerized builds require podman||docker CLI, none found $$PATH' >&2 && exit 1; fi
 	echo BIN_TIMESTAMP==$(BIN_TIMESTAMP)
 	$(CTR_CMD) build -t $(IMAGE_REPO):$(SOURCE_GIT_TAG)-linux-$(ARCH) \
-		-f "$(SRC_ROOT)"/images/build/Dockerfile \
+		-f "$(SRC_ROOT)"/packaging/images/build/Dockerfile \
 		--build-arg SOURCE_GIT_TAG=$(SOURCE_GIT_TAG) \
```

Comment:
I know it wasn't an artifact of this PR but do you mind fixing the indentation on L108-L109?